_posts/2017-4-12-swarm-service-discovery.markdown

```diff
@@ -0,0 +1,185 @@
+---
+layout: post
+title: "Service Discovery under Docker Swarm Mode"
+date:   2017-04-12
+author: "@ajeetsraina"
+tags: [linux,operations]
+categories: beginner
+img: "swarm-service-discovery.png"
+terms: 2
+---
+Service Discovery under Swarm Mode.
+
+## Purpose
+
+The purpose of this lab is to illustrate how Service Discovery works under Swarm Mode.
+
+## The application
+
+WordPress is an open-source content management system (CMS) based on PHP and MySQL. 
+It is a very simple containers application often used for demo purposes during meetup and conferences.
+
+
+## Init your swarm
+
+Let's create a Docker Swarm first
+
+```.term1
+docker swarm init --advertise-addr $(hostname -i)
+```
+
+From the output above, copy the join command (*watch out for newlines*) and paste it in the other terminal.
+
+## Show members of swarm
+
+From the first terminal, check the number of nodes in the swarm.
+
+```.term1
```

Comment:
Maybe remove extra nodes to avoid confusion in the output?. As the one shown by the tutorial will only have two nodes.